src/Core/Enums/ForwardedEmailServiceType.cs

```diff
@@ -4,11 +4,16 @@ namespace Bit.Core.Enums
 {
     public enum ForwardedEmailServiceType
     {
+        None = 0,
         [LocalizableEnum("AnonAddy")]
-        AnonAddy = 0,
+        AnonAddy = 1,
+        [LocalizableEnum("DuckDuckGo")]
+        DuckDuckGo = 2,
+        [LocalizableEnum("Fastmail")]
+        Fastmail = 3,
         [LocalizableEnum("FirefoxRelay")]
-        FirefoxRelay = 1,
+        FirefoxRelay = 4,
         [LocalizableEnum("SimpleLogin")]
-        SimpleLogin = 2,
+        SimpleLogin = 5
     }
```

Comment:
I wouldn't change the values of the types. This configuration gets saved into the device so when an already configured `UsernameGenerationOptions` gets loaded it will have a different type set up.
E.g., let's say the user configured `FirefoxRelay` so an option with type `1` will be saved. But after an app update with this code, when the options are loaded `1` will be `AnonAddy` so the configuration will be wrong.

You either change the values so that they're backwards compatible or add a migration of data.